### PR TITLE
GP2 96 Storybook

### PR DESCRIPTION
**Pull request by csbarbosa**
(Ticket:https://asaphub.atlassian.net/browse/GP2-96)
Added GP2 components into storybook

### CODE REVIEW

File: apps/storybook/src/gp2-components/IconWithLabel.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import projectIcon from '@asap-hub/gp2-components/src/icons/project-icon';
+import IconWithLabel from '@asap-hub/gp2-components/src/molecules/IconWithLabel';
+import { text } from '@storybook/addon-knobs';
+
+export default {
+  title: 'GP2 / Molecules / Icon With Label',
+  component: IconWithLabel,
+};
+export const Logo = () => (
```
Comment (markcsims) [inline apps/storybook/src/gp2-components/IconWithLabel.stories.tsx:9]: Should be Normal
```suggestion
export const Normal = () => (
```

File: apps/storybook/src/gp2-components/HeaderLogo.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export default {
+  title: 'GP2 / Molecules / Header Logo',
+  component: HeaderLogo,
+};
+
+const props = {
+  logoHref: '/',
+};
+
+export const Logo = () => <HeaderLogo {...props} />;
```
Comment (markcsims) [inline apps/storybook/src/gp2-components/HeaderLogo.stories.tsx:12]:
```suggestion
export const Normal = () => <HeaderLogo {...props} />;
```

File: apps/storybook/src/gp2-components/ExpandableText.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  title: 'GP2 / Molecules / Expandable Text',
+  component: ExpandableText,
+};
+
+const props = {
+  description:
+    'In order to show you the Hub, we will need to make your profile public to the Hub network. Would you like to continue?',
+};
+
+export const Text = () => (
```
Comment (markcsims) [inline apps/storybook/src/gp2-components/ExpandableText.stories.tsx:15]:
```suggestion
export const Normal = () => (
```

File: apps/storybook/src/gp2-components/UserCardInfo.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+export default {
+  title: 'GP2 / Molecules / User Card Info',
+  component: UserCardInfo,
+};
+
+type UserCardInfoProps = Pick<ComponentProps<typeof UserRegion>, 'region'> & {
+  workingGroups: { id: string; name: string }[];
+  projects: { id: string; name: string }[];
+  role: string;
+};
```
Comment (markcsims) [inline apps/storybook/src/gp2-components/UserCardInfo.stories.tsx:20]: Looks wrong to be defining the props here. 

maybe we need something like:

```
ComponentProps<typeof UserCardInfo>
```

**Review by markcsims — CHANGES_REQUESTED** [review]

File: apps/storybook/src/gp2-components/IconWithLabel.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  <IconWithLabel icon={projectIcon}>
+    {text(
+      'description',
```
Comment (jhonny-m) [inline apps/storybook/src/gp2-components/IconWithLabel.stories.tsx:12]: I would rename this to `label` and the text could just be "label", doesn't make too much sense being a big text

File: apps/storybook/src/gp2-components/NavigationLink.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+import { text } from '@storybook/addon-knobs';
+
+export default {
+  title: 'GP2 / Molecules / Navigation Link',
+  component: IconWithLabel,
+};
+export const Logo = () => (
+  <NavigationLink href={text('href', 'https://www.google.com')} icon={roleIcon}>
+    {text(
+      'description',
```
Comment (jhonny-m) [inline apps/storybook/src/gp2-components/NavigationLink.stories.tsx:13]: `text` and maybe a "text" as the value

File: apps/storybook/src/gp2-components/UserCardInfo.stories.tsx (hunk 4 above)
Comment (jhonny-m) [reply apps/storybook/src/gp2-components/UserCardInfo.stories.tsx:20]: we shouldn't really need to type cast it tho

Comment (jhonny-m) [reply apps/storybook/src/gp2-components/UserCardInfo.stories.tsx:20]: is there any particular prop that it complains about?

**Review by markcsims — CHANGES_REQUESTED** [review]

**Pull request closed without merging**